src/storage/FileResourceStore.ts

```diff
@@ -300,6 +308,23 @@ export class FileResourceStore implements ResourceStore {
     return quads.concat(containsQuads);
   }
 
+  /**
+   * Helper function to add file system related metadata
+   * @param subject - Subject for the new quads.
+   * @param stats - Stats of the file/directory corresponding to the resource.
+   */
+  private generatePosixQuads(subject: NamedNode, stats: Stats): Quad[] {
+    const quads: Quad[] = [];
+    quads.push(DataFactory.quad(subject, toNamedNode(POSIX.size), toTypedLiteral(stats.size, XSD.integer)));
```

Comment:
```suggestion
    pushQuad(quads, subject, toNamedNode(POSIX.size), toTypedLiteral(stats.size, XSD.integer));
```

with a helper method